### DISCUSSION TITLE
Our NFT doesn't show up in Trustwallet

### DISCUSSION THREAD

**Discussion by FD-Kevn** [trustwallet/wallet-core#2855] [Q&A]
**Describe the bug**

Hello, we are a project called "Frutti Dino," and we have our own "NFTs."

Our website: https://fruttidino.com/
Our Twitter: https://twitter.com/FruttiDino

Our holders have our NFTs in various wallets such as Metamask and Trustwallet.
Our NFTs image was really shown well previously, but suddenly it doesn't show up, so people can not use the "Send" function on mobile TrustWallet. (I attached the images below, how it showed previously)

Our NFT's contract address is as follows
https://bscscan.com/token/0x60ba8ed7cbe095fe7f47b4a70c1d92a2878dae71

We used Openzepplin code for our NFTs
@openzeppelin/contracts-upgradeable/token/ERC721/extensions/ERC721EnumerableUpgradeable.sol

Our NFT's image doesn't show up on mobile
(My wallet address is: 0xCb3F7F25c3d7Bc21b1253daA4bbD052385B233e7)

Please have a close look at this issue.

**Expected behavior**
We would like the Trustwallet's developers to have a look at this issue and make our NFTs visible again

**Screenshots**
![Trust wallet 3](https://user-images.githubusercontent.com/117880023/210919812-6761f8a1-b4c1-4545-8a5d-8cc82c82e7ae.jpg)
![Trustwallet2](https://user-images.githubusercontent.com/117880023/210919827-5e7ee5aa-6926-4d46-a726-c66aff908876.jpg)
![Trust wallet](https://user-images.githubusercontent.com/117880023/210919831-7733f64f-37e6-433e-87aa-2bddac3d582b.jpg)

* These are the previous images that show how our NFTs were shown in Trustwallet! (But now, it's not visible at all)

**Additional context**

**Comment by Milerius** [comment]
Hello @FD-Kevn for end-user application bugs, you can create a ticket here: https://support.trustwallet.com/en/support/home with the category "NFT"

<img width="824" alt="Screenshot 2023-01-06 at 09 33 55" src="https://user-images.githubusercontent.com/21139416/210962782-78c292a7-44a7-4c73-94c8-771d09c74735.png">

I wish you a great day!

**Reply by FD-Kevn** [reply]
@Milerius 
We just submitted the ticket. Please have a look and solve the problem.
Thank you.
